<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hm3u8dl_gui</class>
 <widget class="QWidget" name="hm3u8dl_gui">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>492</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>hm3u8dl_gui</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhLowercaseOnly|Qt::ImhPreferLowercase|Qt::ImhPreferUppercase</set>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>77</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_update">
      <property name="text">
       <string>检查更新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_workDir">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>451</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_enableDel">
      <property name="text">
       <string>删除分片</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_mergeMode">
      <property name="text">
       <string>合并方式</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_mergeMode">
      <property name="currentText">
       <string>3</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_thread">
      <property name="text">
       <string>线程数</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEdit_thread">
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_port">
        <property name="text">
         <string>8000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>291</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_verison">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_workDir">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>255</y>
     <width>371</width>
     <height>26</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_key">
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这是key&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_key">
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AES-128-CBC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AES-128-ECB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SAMPLE-AES-CTR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SAMPLE-AES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cbcs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>copyrightDRM</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>292</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLineEdit" name="lineEdit_nonce"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_iv"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_proxy"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>22</y>
     <width>78</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_verison">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>版本号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_workDir">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>工作目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_m3u8url">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>m3u8地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_headers">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>headers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_baseurl">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>baseurl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_key">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_nonce">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>nonce</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_iv">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>iv</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_proxy">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>代理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>97</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="lineEdit_m3u8url"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_title"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_headers"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_baseurl"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="lineEdit_data"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Start!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
